### DISCUSSION TITLE
Sort column of MasterDetailRelation

### DISCUSSION THREAD

**Discussion by patriciomh1965** [serenity-is/Serenity#5645] [Help]
Hello Community

I need to sort the columns of the grid, is there any way to do it:

**TaMstCuadraturaRow
-----------------------**
...,
....

        [DisplayName(""), MasterDetailRelation(foreignKey: "vou_prc_control", MasterKeyField = "res_control"), NotMapped]
        [MinSelectLevel(SelectLevel.List)]
        public List<TaMstCuadraturaDetalleRow> DetailCuadratura 
        {
            get { return Fields.DetailCuadratura[this]; }
            set { Fields.DetailCuadratura[this] = value; }

**TaMstCuadraturaDetalleRow
-------------------------------**
namespace PLAINT.ServiciosCnc.Entities
{
    using PLAINT.Modules.ServiciosCnc;
    using Serenity;
    using Serenity.ComponentModel;
    using Serenity.Data;
    using Serenity.Data.Mapping;
    using System;
    using System.ComponentModel;
    using System.IO;

    [ConnectionKey("ServiciosCnc"), Module("ServiciosCnc"), TableName("[dbo].[vw_web_lista_cuadratura_contable]")]
    [DisplayName("Cuadratura - Detalle"), InstanceName("Cuadratura - Detalle")]
    [ReadPermission("Administration:General")]
    [ModifyPermission("Administration:General")]

    [LookupScript(Permission = "*")] // VINCULACION
    public sealed class TaMstCuadraturaDetalleRow : Row, IIdRow, INameRow
    {
        [DisplayName("ID"), Column("vou_id"), Identity]
        public Int64? IdVou
        {
            get { return Fields.IdVou[this]; }
            set { Fields.IdVou[this] = value; }
        }

Attached image
![Imagen](https://user-images.githubusercontent.com/42218266/115171323-cec9d280-a090-11eb-8b65-6526b9c2f97e.png)

I will appreciate the help you can give

**Comment by minhhungit** [comment]
@brunobola implemented it before here https://github.com/serenity-is/Serenity/issues/3792, please using github search before asking, I found many solutions for myself by searching on github

**Comment by patriciomh1965** [comment]
Hi jin
Add the instructions to the grid editor. All good, but I can't filter
the information. I attach the images and the ts.
It would be a great help if you tell me where my problem is, I have
reviewed all the issues but nothing.

That my friend.



Thank you
